code/modules/clothing/suits/wiz_robe.dm

```diff
@@ -183,7 +183,7 @@
 
 /obj/item/clothing/suit/space/hardsuit/shielded/wizard
```

Comment:
This is going to need repathing to not be a suit/space/hardsuit subtype

Reply:
@Takahiru